documentation/general/program-scale-testing-requirements-draft.md

```diff
@@ -204,7 +204,7 @@ integrators to track and test the designs in a common manner.
         seen that meets the policy for the bi-directional flow setup.
         When using this in CPS testing, we will send a total of 6
         packets to match TCP to make things more balanced. UDP
-        bi-directional flows will be aged within 1 second after
+        bi-directional flows will be aged within 5 second after
         receiving the last packet. I would set all UDP bi-directional
```

Comment:
Should the last sentence be removed ("0.5 to 1.0 second aging")?

Reply:
corrected to 5 seconds

Reply:
LGTM, thank you Mircea for changing ENI to 32 and CPS to 3M.